prov/shm/src/smr_init.c

```diff
@@ -44,6 +44,7 @@ struct smr_env smr_env = {
 	.sar_threshold = SIZE_MAX,
 	.disable_cma = false,
 	.use_dsa_sar = false,
+	.max_gdrcopy_size = 3072,
```

Comment:
Can it be overridden by  `fi_param_get`  ?

Reply:
Currently not. The configuration is not exposed  yet.

Reply:
Yeah, I am asking if there is a follow-up PR on it.

Reply:
I see. I don't plan to make it configurable at this point. My experiments are based on p4d only and I don't see supporting data to change it at this point.

Reply:
are other variables in smr_env configurable?

Reply:
Yes. I don't think that's necessary though.

Reply:
I think we should make it configurable too. This allows benefits for future experiments without the need of rebuilding the code. Like all the variables in rxr_env.

Reply:
Actually we do have variables in rxr_env that are not configurable from CLI.

Introducing the environment variable can be left to the future IMO. It is easy to add new variables than removing old ones.